mappings/net/minecraft/util/dynamic/Codecs.mapping

```diff
@@ -18,25 +20,93 @@ CLASS net/minecraft/unmapped/C_tyjpezxh net/minecraft/util/dynamic/Codecs
+		ARG 1 collection
 	METHOD m_ceufsyxl withLifecycle (Lcom/mojang/serialization/Codec;Ljava/util/function/Function;Ljava/util/function/Function;)Lcom/mojang/serialization/Codec;
 		ARG 0 codec
 		ARG 1 applyFunction
 		ARG 2 coApplyFunction
+	METHOD m_cjkdnnjx (Ljava/lang/String;)Lcom/mojang/serialization/DataResult;
+		ARG 0 string
+	METHOD m_cjqijzpr createStrictOptionalField (Lcom/mojang/serialization/Codec;Ljava/lang/String;)Lcom/mojang/serialization/MapCodec;
+		ARG 1 name
+	METHOD m_cqinaoub (IILjava/util/function/Function;Ljava/lang/Integer;)Lcom/mojang/serialization/DataResult;
+		ARG 3 integer
+	METHOD m_dgtnqbdd withOptionalParameters (Ljava/lang/String;Ljava/lang/String;Lcom/mojang/serialization/Codec;Ljava/util/function/Function;Ljava/util/function/Function;)Lcom/mojang/serialization/MapCodec;
+		ARG 0 typeKey
+		ARG 1 parametersKey
+		ARG 2 typeCodec
+		ARG 3 typeGetter
+		ARG 4 parameterCodecGetter
+	METHOD m_dmqewsll (Lorg/joml/Quaternionf;)Ljava/util/List;
+		ARG 0 quaternionf
+	METHOD m_dnfhmcbw (Ljava/util/function/Function;Lcom/mojang/datafixers/util/Either;)Ljava/lang/Object;
+		ARG 1 either
 	METHOD m_dujfhivb orElsePartial (Ljava/lang/Object;)Lcom/mojang/serialization/Codec$ResultFunction;
 		ARG 0 partial
+	METHOD m_eypcvcqq (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
+		ARG 0 instance
+	METHOD m_flhgmmki fromOps (Lcom/mojang/serialization/DynamicOps;)Lcom/mojang/serialization/Codec;
+	METHOD m_fvpeuyuk (Ljava/lang/String;)Lcom/mojang/serialization/DataResult;
+		ARG 0 string
+	METHOD m_gcywmtgx (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
+		ARG 0 instance
+	METHOD m_gglkhqew (Ljava/util/List;)Lorg/joml/Matrix4f;
+		ARG 0 list1
+	METHOD m_gtuotoel createStrictUnboundedMap (Lcom/mojang/serialization/Codec;Lcom/mojang/serialization/Codec;)Lnet/minecraft/unmapped/C_tyjpezxh$C_psivvwhx;
+		ARG 0 keyCodec
+		ARG 1 elementCodec
+	METHOD m_gyrtopxy (Ljava/lang/String;)Lcom/mojang/serialization/DataResult;
+		ARG 0 string
+	METHOD m_gywbxzse (Lcom/mojang/serialization/Codec;Ljava/lang/String;Ljava/lang/String;Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
+		ARG 3 instance
+	METHOD m_hpsmlhqm (Ljava/util/List;)Lcom/mojang/serialization/DataResult;
+		ARG 0 list
+	METHOD m_hsukemqs (Ljava/util/function/IntFunction;Ljava/lang/Integer;)Lcom/mojang/serialization/DataResult;
+		ARG 1 id
+	METHOD m_htlmmuvh (Ljava/lang/Object;)Ljava/lang/Object;
+		ARG 0 object
+	METHOD m_ifdvuwzy formattedTime (Ljava/time/format/DateTimeFormatter;)Lcom/mojang/serialization/Codec;
+		ARG 0 formatter
+	METHOD m_ihmqisju (FFLjava/util/function/Function;Ljava/lang/Float;)Lcom/mojang/serialization/DataResult;
+		ARG 3 f
 	METHOD m_ihvqsgya retrieveContext (Ljava/util/function/Function;)Lcom/mojang/serialization/MapCodec;
 		ARG 0 retriever
+	METHOD m_iuewvnnu ([B)Ljava/lang/String;
+		ARG 0 bs
```

Comment:
```suggestion
		ARG 0 bytes
```
this looked like B.S. more than the plural of `b` to me, I think `bytes` is clearer